package.json

```diff
@@ -68,7 +68,6 @@
     "intersection-observer": "0.4.3",
     "pepjs": "0.4.3",
     "tslib": "~1.8.1",
-    "web-animations-js": "2.3.1",
```

Comment:
Should this still be a dependency? intersection and pepjs are also handled by `browser.ts` from shim but they are still dependencies or should they be removed from here? Noticed that the versions are pinned here but in shim they are using `^`